### PR TITLE
Tweak test suite (duration, centos:8, datadir)

### PR DESCRIPTION
**Pull request by lukaszachy**
Extend duration to accomodate slow dnf cache updates
Also typo in fedpkg command

I'm still running this in Beaker so lets see if it PASSES

### CODE REVIEW

**Comment by lukaszachy** [comment]
One issue still present - /tests/run/worktree/ for data/prepare
But we have difference between -h container and -h virtual . In container is not necessary to copy to TMT_PLAN_DATA.

File: tests/run/worktree/data/prepare/plan.fmf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 provision:
     how: container
 prepare:
-    script: echo content > file
+    script: echo content > $TMT_PLAN_DATA/file
 execute:
-    script: grep content file
+    script: grep content $TMT_PLAN_DATA/file
```
Comment (psss) [inline tests/run/worktree/data/prepare/plan.fmf:6]: I'm not sure about this change. Preparing a file during the `prepare` step and then using it during `execute` without need to `pull()` it from the guest is a valid use case. I'm afraid this test failing with the `virtual` provision is just confirming #964.

**Review by psss — CHANGES_REQUESTED** [review]
Thanks for all the updates. Everything looks good except for the `/tests/run/worktree` adjustment. It seems to hide #964 which is still not fixed.

File: tests/run/worktree/data/prepare/plan.fmf (hunk 1 above)
Comment (lukaszachy) [reply tests/run/worktree/data/prepare/plan.fmf:6]: OK. So we can keep test as is to check for #964

**Review by psss — APPROVED** [review]
Looks good!

**Pull request merged**